mlir/utils/performance/perfRunner.py

```diff
@@ -129,6 +139,9 @@ def create_paths(config_file_path, mlir_build_dir_path) -> Paths:
 
 # utility functions.
 def getNanoSeconds(fileName):
+    pass
```

Comment:
I'm going to assign V1 or V2 to getNanoSeconds.  I don't really need this "pass" implementation.